ansible/roles/oso_console_extensions/templates/console-extensions-template.yml.j2

```diff
@@ -0,0 +1,139 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: online-console-extensions
+
+parameters:
+- name: NAME
+  description: The name of the DeploymentConfig
+  value: online-console-extensions
+- name: GIT_REPO
+  description: Git repository housing the code to build and deploy
+  value: "https://github.com/openshift/online-console-extensions.git"
+- name: GIT_REF
+  description: Git branch, commit, or tag to deploy
+  value: "master"
+- name: TARGET
+  description: One of free|paid
+  value: "{{ oso_extensions_target }}"
+
+objects:
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: ${NAME}
+    labels:
+      app: ${NAME}
+  spec:
+    source:
+      git:
+        uri: ${GIT_REPO}
+        ref: ${GIT_REF}
+    strategy:
+      sourceStrategy:
+        env:
+        - name: TARGET
+          value: ${TARGET}
+        from:
+          kind: DockerImage
+          name: ${NAME}-builder:latest
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ${NAME}:latest
+    triggers:
+      - type: ConfigChange
+
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: ${NAME}
+    labels:
+      app: ${NAME}
+  spec:
+  status:
+    dockerImageRepository:
+
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    name: ${NAME}
+    labels:
+      app: ${NAME}
+  spec:
+    strategy:
+      type: Recreate
+    triggers:
+      - type: ConfigChange
+      - imageChangeParams:
+          automatic: true
+          containerNames:
+          - ${NAME}
+          from:
+            kind: ImageStreamTag
+            name: ${NAME}:latest
+            namespace: test
+        type: ImageChange
+        imageChangeParams:
```

Comment:
You have two `imageChangeParams` stanzas for the same trigger (only difference being that the first one has `.from.namespace`).

Reply:
@Miciah thanks for reviewing this!! Much needed